### DISCUSSION TITLE
How to make key binding M4 change GPU mode?

### DISCUSSION THREAD

**Discussion by NickTar87** [seerge/g-helper#211] [Q&A]
I want to set up the custom key bindings so that M4 (the ROG button) changes the GPU mode between Eco and Optimized. This way I can control CPU performance and fan speeds using the regular performance button (fan button) and GPU modes with the button right next to it.

**Comment by seerge** [comment]
You can't set that on a keybind. And to be honest - switching dGPU off is quite invasive action (that may stop dGPU using apps). Why would you want to do it that often that you want a hotkey ?

**Reply by NickTar87** [reply]
I frequently go between playing games on the dGPU and using my laptop for schoolwork or videos. It would be nice to have a single key to switch instead of using the popup. Same reason as the laptops have a hotkey for the different CPU and fan modes.
